libs/domain/user/src/services/address.service.ts

```diff
@@ -4,7 +4,7 @@ import { Address } from '../models';
 export interface AddressService {
   getCurrentAddress(): Observable<Address | null>;
   getAddress(addressId: string): Observable<Address | null>;
-  getAddresses(): Observable<Address[] | null>;
+  getAddresses(): Observable<Address[] | undefined>;
```

Comment:
What about following CRUD?
What about avoiding the domain in the methods?

```ts
create(data: Address): Observable<Address | null>;
read(data: Address): Observable<Address | null>;
update(data: Address): Observable<Address | null>;
delete(data: Address): Observable<Address | null>;

//
readAll(): Observable<Address[] | null | undefined>; 
(or a single read with conditional args)
readCurrent(): Observable<Address | null>;
or
current(): Observable<Address | null>;
```

Reply:
👍